frontend/src/lib/router.tsx

```diff
@@ -673,7 +673,23 @@ const defaultRoutes: {
     exact: true,
     name: 'Settings',
     sidebar: {
-      item: 'settings',
+      item: 'general',
+      sidebar: DefaultSidebars.HOME,
+    },
+    useClusterURL: false,
+    noAuthRequired: true,
+    component: () => (
+      <PageGrid>
+        <Settings />
+      </PageGrid>
+    ),
+  },
+  settingsGeneral: {
+    path: '/settings/general',
+    exact: true,
+    name: 'General Settings',
+    sidebar: {
+      item: 'general',
```

Comment:
This one I think we don't need because we already have /settings, right?

Reply:
Yup, removed